ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/ProtocolScheduleBuilder.java

```diff
@@ -39,36 +40,41 @@
   private final boolean isRevertReasonEnabled;
   private final BadBlockManager badBlockManager = new BadBlockManager();
   private final boolean quorumCompatibilityMode;
+  private final PowAlgorithm powAlgorithm;
 
   public ProtocolScheduleBuilder(
       final GenesisConfigOptions config,
       final BigInteger defaultChainId,
       final Function<ProtocolSpecBuilder, ProtocolSpecBuilder> protocolSpecAdapter,
       final PrivacyParameters privacyParameters,
       final boolean isRevertReasonEnabled,
-      final boolean quorumCompatibilityMode) {
+      final boolean quorumCompatibilityMode,
+      final PowAlgorithm powAlgorithm) {
```

Comment:
Pow Algorithm is an aspect of the fork, not an externally configured element.  It should not be part of these constructors, nor passed into the child fork definitions.